core/templates/App.ts

```diff
@@ -25,7 +25,6 @@
  */
 import { OppiaAngularRootComponent } from 'components/oppia-angular-root.component';
 import 'firebase/auth';
-import 'hammerjs';
```

Comment:
You can remove `hammerjs` from _package.json_ right?

Reply:
Used in shared-components.module.ts